var/spack/repos/builtin/packages/neurodamus-model/package.py

```diff
@@ -157,13 +157,11 @@ def install(self, spec, prefix):
                 prefix.bin.special)
 
         if spec.satisfies('+coreneuron'):
-            # move special-core to bin and libcorenrnmech* to lib
             outdir = spec.architecture.target
-            shutil.move(join_path(outdir, 'special-core'), prefix.bin)
-            for libname in find_libraries("libcorenrnmech*", outdir):
-                shutil.move(libname, prefix.lib)
+            install = which('nrnivmech_install.sh', path=".")
+            install(prefix)
             # Then modc
-            shutil.move(join_path(outdir, 'modc_core'), prefix.share)
+            shutil.move(join_path(outdir, 'core/mod2c'), prefix.share.modc_core)
```

Comment:
@ferdonline  : I see coreneuron variant not enabled in any new packages:

```
variant('coreneuron',  default=False, description="Enable CoreNEURON Support")
```

shouldn't this be enabled in neurodamus-neocortex at least?

Reply:
'+coreneuron' variant is inherited by all models. Maybe we it is possible to change the default only in neocortex, but I find somewhat better to keep all the same and dependencies minimal so that users can easily install it. For deployment we definitely should have a (single?) version supporting coreneuron
But if you see other reasons to have it on by default I can look into changing it.

Reply:
agree, its fine for now...we can fix the deployment separately.